tlf.1.in

```diff
@@ -1412,160 +2057,379 @@ Show the country/call check window.
 .B PARTIALS
 Show a list of possible contest calls.
 .
+.IP
+See
+.B CALLMASTER
+in the
+.B RULES
+section below for partials file specification.
+.
 .TP
 .B USEPARTIALS
-Use the auto-complete utility (takes some practice...).  @PACKAGE_NAME@ will
-complete the call as soon as it is unique in the database.  This can of course
-lead to strange effects, but in practice there are far more hits than misses.
-Sometimes you must edit the call because it has locked on a unique call.  Try
-it, and switch it off when you don't like it.
+Use the auto-complete utility (takes some practice...).
+.
+.IP
+@PACKAGE_NAME@ will complete the call as soon as it is unique in the database.
+.
+.IP
+This can lead to strange effects, but in practice there are far more hits than
+misses.
+.
+Sometimes you must edit the call because it has locked on a unique call.
+.
+Try it, and switch it off when you don't like it.
 .
 .TP
 .B LOGFREQUENCY
-Put frequency (kHz) into QSO number to enable logging of frequency (only QSO
-and dxped mode).
+Put frequency (kHz) into QSO number to enable logging of frequency (only in
+.B QSO
+or
+.B DXPED
+modes).
 .
 .TP
 .B IGNOREDUPES
```

Comment:
Just came across this: it's spelled in singular, `IGNOREDUPE`.

Reply:
Good catch. Fixed.